worktree.go

```diff
@@ -26,11 +26,12 @@ import (
 )
 
 var (
-	ErrWorktreeNotClean     = errors.New("worktree is not clean")
-	ErrSubmoduleNotFound    = errors.New("submodule not found")
-	ErrUnstagedChanges      = errors.New("worktree contains unstaged changes")
-	ErrGitModulesSymlink    = errors.New(gitmodulesFile + " is a symlink")
-	ErrNonFastForwardUpdate = errors.New("non-fast-forward update")
+	ErrWorktreeNotClean                 = errors.New("worktree is not clean")
+	ErrSubmoduleNotFound                = errors.New("submodule not found")
+	ErrUnstagedChanges                  = errors.New("worktree contains unstaged changes")
+	ErrGitModulesSymlink                = errors.New(gitmodulesFile + " is a symlink")
+	ErrNonFastForwardUpdate             = errors.New("non-fast-forward update")
+	ErrRestoreWorktreeeOnlyNotSupported = errors.New("--worktree only is not supported")
```

Comment:
Remove the `--`

Reply:
Commit a1221f4cf9ef4bc108fadfc98fcb69823f1109df